config/file_config.go

```diff
@@ -295,6 +295,15 @@ func (f *fileConfig) validateConditionalConfigs() error {
 			return err
 		}
 	}
+
+	// validate cache strategy
+	st := f.GetCacheOverrunStrategy()
+	switch st {
+	case "resize", "impact":
+		break
+	default:
+		return fmt.Errorf("invalid CacheOverrunStrategy: '%s'", st)
```

Comment:
An empty value `""` would cause this to error.

What happens when an error is returned here? Does it stop refinery from starting?

Reply:
Because a default of "resize" is baked into the code, an empty value is only possible if someone explicitly overrides the configuration and forces it to empty, or some other value we don't recognize. 

In these cases, if it occurs on initial config load, refinery logs it and fails to start -- if it occurs on an edit to a running config, it logs it and doesn't change the loaded config.